python/cudf_polars/tests/expressions/test_datetime_basic.py

```diff
@@ -37,24 +40,105 @@ def test_datetime_dataframe_scan(dtype):
     assert_gpu_result_equal(query)
 
 
+datetime_extract_fields = [
+    "year",
+    "month",
+    "day",
+    "weekday",
+    "hour",
+    "minute",
+    "second",
+    "millisecond",
+    "microsecond",
+    "nanosecond",
+]
+
+unsupported_extract_fields = ["millennium"]
```

Comment:
I guess we could extract this from years rather than not support it…

Reply:
This was a rough solution for maintaining 100% coverage of the code, specifically getting us to hit the error line that complains about a rust node that is exposed but that we don't support. Because of this, adding `millennium` would in some sense just kick the can down the road until we come to a day where all the rust nodes are supported in python as well and we can get rid of the error.

Instead of doing that I went with an approach that we liked in a similar situation where we monkeypatch things to trigger an error at the desired location, regardless of what we support. See the latest commits for details and let me know what you think.